packages/embeds/vite.config.js

```diff
@@ -6,6 +6,8 @@ process.env.EMBED_PUBLIC_WEBAPP_URL = process.env.NEXT_PUBLIC_WEBAPP_URL;
 process.env.EMBED_PUBLIC_EMBED_LIB_URL = process.env.NEXT_PUBLIC_EMBED_LIB_URL;
 // eslint-disable-next-line turbo/no-undeclared-env-vars
 process.env.EMBED_PUBLIC_EMBED_FINGER_PRINT = process.env.NEXT_PUBLIC_EMBED_FINGER_PRINT;
+// eslint-disable-next-line turbo/no-undeclared-env-vars
+process.env.EMBED_PUBLIC_EMBED_VERSION = process.env.NEXT_PUBLIC_EMBED_VERSION;
```

Comment:
Following in the footprints of `EMBED_PUBLIC_EMBED_FINGER_PRINT` to add this new env variable.